cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceHandler.java

```diff
@@ -346,15 +341,14 @@ private boolean dropDataset(Id.DatasetInstance datasetInstanceId, DatasetSpecifi
     return true;
   }
 
-  // TODO: Needs to be namespaced
-  private void disableExplore(String name) {
+  private void disableExplore(Id.DatasetInstance datasetInstance) {
     // Disable ad-hoc exploration of dataset
     // Note: today explore enable is not transactional with dataset create - CDAP-8
     try {
-      exploreFacade.disableExploreDataset(name);
+      exploreFacade.disableExploreDataset(datasetInstance);
     } catch (Exception e) {
       String msg = String.format("Cannot disable exploration of dataset instance %s: %s",
-                                 name, e.getMessage());
+                                 datasetInstance.getId(), e.getMessage());
```

Comment:
May be worth it to not to a ``getId`` here, so both namespace and instance name get logged?